shard/init.lua

```diff
@@ -2104,6 +2123,33 @@ local function init(cfg, callback)
     return true
 end
 
+-- background fiber function that signals when all shards are connected
+local function wait_connection()
```

Comment:
I still do not like, that there are wait_connection and wait_connected. Names are almost identical.